### ISSUE TITLE
Use Elmo to build a model 

### ISSUE THREAD

**Issue by sehsanm** [sehsanm/embedding-benchmark#25]

**Comment by abb4s** [comment]
I want to assign it to myself . but I don't know how.

**Comment by sehsanm** [comment]
Hi
You cannot start this task unless our corpora are ready. Please move it back to TODO and unassign it

**Comment by sehsanm** [comment]
Have you started ? 
Are you going to work on Wiki dump ?

**Comment by roetezadi** [comment]
Yes, I have read that but I can't download it!  it seems that I can't access to sbuacir-my.sharepoint.com directory! this is the issue: Issue Type: User not in the directory.

**Comment by sehsanm** [comment]
please try this :

https://sbuacir-my.sharepoint.com/:f:/g/personal/se_mahmoudi_sbu_ac_ir/EtCWTI-YEoRLiA3G3GtZOPQB-Qx3VL1hnquYlIUPez3-pA?e=fnU0Me

**Comment by roetezadi** [comment]
Nothing is in there! it's just a white page with your name in it and a "feedback" button :D

**Comment by roetezadi** [comment]
Yes, I see it now thanks! it has 4 parts

**Comment by sehsanm** [comment]
Hi  I cannot see you model ? Have you uploaded it ? I'm passing the scores toady
